packages/components/tests/dummy/app/templates/components/tabs.hbs

```diff
@@ -52,14 +54,23 @@
   <dl class="dummy-component-props" aria-labelledby="component-api-tab">
     <dt>count <code>string</code></dt>
     <dd>
-      <p>Displays an optional count in the Tab.</p>
-      <p>Accepts the text value that should go in the badge counter.</p>
+      <p>
+        Displays an optional
+        <a href="/components/badge-count" target="_blank" rel="noopener noreferrer">BadgeCount</a>
+        in the Tab.
+      </p>
+      <p>
+        Accepts the text value that should go in the
+        <a href="/components/badge-count" target="_blank" rel="noopener noreferrer">BadgeCount</a>.
+      </p>
```

Comment:
probably I would avoid the duplication/repetition doing something like this:
```suggestion
      <p>Displays an optional "count" indicator in the tab. Accepts the text value that should go in the
        <a href="/components/badge-count">BadgeCount</a>.</p>
```
(notice: since it's the same website, we don't need `target="_blank"`)

Reply:
Thanks. I didn't know if you would prefer to open links like these within the same tab or a new one. Will just open in the same one in the future.